capstone-ui/src/components/ProfileCard/ProfileCard.jsx

```diff
@@ -1,24 +1,48 @@
 import React from "react";
 import "./ProfileCard.css";
+import SearchPage from "../SearchPage/SearchPage";
+import axios from "axios";
+import * as config from "../../config";
+import { createRef } from "react";
+
+const ProfileCard = ({ handleLogin }) => {
+  const username = React.createRef();
+  const password = React.createRef();
+
+  const handleSubmit = (event) => {
```

Comment:
Seems like handleSubmit is never used here. Try to not keep unused functions and variables in the code.